testing/tests/ui/macro-args-hashtag.rs

```diff
@@ -0,0 +1,20 @@
+// This test ensure that we have the right error if a `#` character isn't prepended by
+// a whitespace character.
+
+use askama::Template;
+
+#[derive(Template)]
+#[template(
+    source = r###"{{z!{'r#}}}"###,
+    ext = "html"
+)]
+struct Example;
+
+#[derive(Template)]
+#[template(
+    source = r###"{{z!{#r' y}}}"###,
```

Comment:
```suggestion
    source = r###"{{z!{'r# y}}}"###,
```

Reply:
Makes more sense, was wondering if it was intended. ^^'

Reply:
Well, there's six ways to order three different letters: `'r#`, `'#r`, `r'#`, `r#'`, `#r'`, `r#'`. That means that I beat the odds when I type them in the correct order half of the time! :D 🙈